src/compiler/compile/css/Selector.ts

```diff
@@ -160,7 +160,7 @@ function apply_selector(blocks: Block[], node: Element, stack: Element[], to_enc
 			return false;
 
 		case BlockAppliesToNode.UnknownSelectorType:
-			// bail. TODO figure out what these could be
+		// bail. TODO figure out what these could be
```

Comment:
this doesn't look right to me

Reply:
The "correct" format for a switch statement is un-indented, so I've gone with that with the view to vote on our preferred indentation format, since both styles were present in the codebase, from what I remember.

Reply:
The thing that's unexpected to me here is that it's changing only the indentation of the comment and not the code

I prefer the indented `switch` statement for what it's worth. I think all the code here and in Sapper is indented (I just did a quick `grep -A 2 -r case src/`). The only exception would be single-line `case` statements like `case 'AwaitBlock': return AwaitBlock;`

Reply:
I also prefer indented, but the goal was to not change any rules at all (where possible). New rules should be voted on via the PR system on the eslint-config project.